drivers/SmartThings/zigbee-button/src/zigbee-multi-button/supported_values.lua

```diff
@@ -17,30 +17,38 @@ local devices = {
     MATCHING_MATRIX = {
       { mfr = "IKEA of Sweden", model = "TRADFRI on/off switch" }
     },
-    SUPPORTED_BUTTON_VALUES = {"pushed", "held"},
+    SUPPORTED_BUTTON_VALUES = { "pushed", "held" },
     NUMBER_OF_BUTTONS = 2
   },
   BUTTON_PUSH_HELD_4 = {
     MATCHING_MATRIX = {
       { mfr = "CentraLite", model = "3450-L" },
-      { mfr = "CentraLite", model = "3450-L2" }
+      { mfr = "CentraLite", model = "3450-L2" },
+      { mfr = "ROBB smarrt", model = "ROB_200-008-0" }
     },
-    SUPPORTED_BUTTON_VALUES = {"pushed", "held"},
+    SUPPORTED_BUTTON_VALUES = { "pushed", "held" },
     NUMBER_OF_BUTTONS = 4
   },
+  BUTTON_PUSH_HELD_8 = {
+    MATCHING_MATRIX = {
+      { mfr = "ROBB smarrt", model = "ROB_200-007-0" },
+    },
+    SUPPORTED_BUTTON_VALUES = { "pushed", "held" },
+    NUMBER_OF_BUTTONS = 8
+  },
```

Comment:
since you're not using any of the code you added here (and it lists the wrong events anyway (held instead of up_held and down_held) please remove it

Reply:
solved with 714d055